core/templates/pages/exploration-player-page/templates/display-hint-modal.controller.spec.ts

```diff
@@ -57,7 +57,14 @@ describe('Display Hint Modal Controller', function() {
 
   importAllAngularServices();
 
-  beforeEach(angular.mock.module('oppia'));
+  beforeEach(angular.mock.module('oppia', function($provide) {
+    $provide.value('NgZone', function() {
+      this.runOutsideAngular = function(callback) {
+        callback();
+      };
+    });
+  }));
+
```

Comment:
Why are we providing it here? I don't see any changes to display-hint-modal.

Reply:
done